inferno-framework/tutorial-test-kit
# FI-3166: Step 5 solution upgrade

## Summary
This PR updates the Inferno template version in step-5-solution branch. It includes the solution to step-5 of the tutorial.

## Testing
Run locally with Docker:
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - access_token: `SAMPLE_TOKEN`
    - Patient ID: `85`
- You should get the following results:

![image](https://github.com/user-attachments/assets/8fa9ad8e-1dba-4462-87f5-09c936379e53)

- And more specifically, the following results for the Search Tests group:
![image](https://github.com/user-attachments/assets/c1931ead-50b8-402a-8dd1-81c5499f778a)

NOTE: The 2.02 Test fails because the Patients in the reference server no longer conform to USCore 3.1.1. This is expected.

Also run with Ruby.
